LarkatorGUI/MainWindow.xaml.cs

```diff
@@ -729,18 +768,33 @@ private void UpdateSearchResults(IList<SearchCriteria> searches)
 
                 ListResults.Clear();
                 foreach (var result in found)
-                    ListResults.Add(result);
+                    if (!Properties.Settings.Default.hideUntameable || (result.IsTameable))
+                        ListResults.Add(result);
 
                 ShowCounts = true;
                 ResultTotalCount = ShowTames ? sourceDinos.Sum(species => species.Value.Count()) : total;
                 ResultMatchingCount = ListResults.Count;
+
+                adjustSearchColumns();
+
             }
 
             ((CollectionViewSource)Resources["OrderedResults"]).View.Refresh();
 
             TriggerNameSearch(true);
         }
 
+        private void adjustSearchColumns()
```

Comment:
Fixed the camel case name in commit 8d19102d95ef98d5c22d51e471bc8738995626d3.

I ended up switching to the method because I couldn't get the binding to work correctly, I forgot to switch back to the the binding method before commiting, so I can implement that shortly.

Reply:
Removed the Adjust function and switched to bindings in commit 2f3882ae763aa8a68d0a541e46e37263572f985b

For some reason Visual Studio flags the binding, but it still works without any issues I can see.